lib/ex_rabbitmq/rpc/server.ex

```diff
@@ -222,8 +223,12 @@ defmodule ExRabbitMQ.RPC.Server do
             xrmq_basic_ack(delivery_tag, state)
             {:noreply, state}
 
+          {:requeue, state} ->
```

Comment:
Why not change `:requeue` to `:reject_requeue`?

As per https://www.rabbitmq.com/nack.html, there are other ways to requeue as well.